samples/nrf_desktop/src/events/config_event.h

```diff
@@ -30,7 +30,9 @@ enum config_event_id {
 	CONFIG_EVENT_ID_MOUSE_CPI,
 	CONFIG_EVENT_ID_MOUSE_DOWNSHIFT_RUN,
 	CONFIG_EVENT_ID_MOUSE_DOWNSHIFT_REST1,
-	CONFIG_EVENT_ID_MOUSE_DOWNSHIFT_REST2
+	CONFIG_EVENT_ID_MOUSE_DOWNSHIFT_REST2,
+
+	CONFIG_EVENT_ID_MAX
```

Comment:
as commented in other place - these enums must have explicit values that match with definition in the tool
e.g.
```
enum ids {
   id_reserved = 0,
   id_1 = 1,
   id_2 = 2,
...
   id_count,
};
```

Reply:
done